electrum/lnworker.py

```diff
@@ -2043,7 +2043,7 @@ async def create_routes_for_payment(
                         self.logger.info(f'per trampoline fees: {per_trampoline_fees}')
                         for chan_id, part_amount_msat in trampoline_parts:
                             chan = self.channels[chan_id]
-                            margin = chan.available_to_spend(LOCAL, strict=True) - part_amount_msat
+                            margin = chan.available_to_spend(LOCAL) - part_amount_msat
```

Comment:
In relation to my other comment, we could specify `single_payment=True` here, and perhaps in this case it might be sufficient. (if we don't split a payment into multiple parts within a given channel)

Reply:
Indeed I  think it should be specified here. This branch is for the trampoline case, where we do not split a payment within a given channel.